cns/grpc/proto/server.pb.go

```diff
@@ -0,0 +1,409 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Like we discussed, we want to put all `protoc` -generated Go code in a seperate `protogen` directory, so let's remove this file

Reply:
I have removed the file and git ignored it